doc/nrf/ug_dev_model.rst

```diff
@@ -0,0 +1,514 @@
+.. _dev-model:
+
+|NCS| development model
+#######################
+
+This page describes the development model that the |NCS| follows in order to
+give its users access to both stable releases and also the bleeding edge.
+
+Please refer to the :ref:`dm-glossary` for additional clarification on terms
+used in these sections.
+
+Overview
+********
+
+The |NCS| is an SDK developed and maintained by Nordic Semiconductor to enable
+development of applications designed to run on its ICs (Integrated Circuits). It
+is based on both code developed by Nordic engineers and on open source software
+projects, which in turn are developed collaboratively among many contributors
+around the world.
+The |NCS| is developed openly, and its users can track the development progress
```

Comment:
I've asked before, but I forget your answer: do you really mean to be adding single newlines here? You need two newlines to separate paragraphs, so this looks a bit weird.

Reply:
Did you mean to leave the single newline after "portions of the SDK"? Same issue there.

Reply:
oversight, thanks!